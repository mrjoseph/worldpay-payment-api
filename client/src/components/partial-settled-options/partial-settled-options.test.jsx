import React from 'react';
import renderer from 'react-test-renderer';
import PartialSettledOptions from './partial-settled-options';

describe('CancelledOptions', () => {
  it('Should render the breadcrumb with props', () => {
    const rendered = renderer.create(
      <PartialSettledOptions />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
