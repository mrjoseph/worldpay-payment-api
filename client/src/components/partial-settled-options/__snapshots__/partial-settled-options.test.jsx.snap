// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelledOptions Should render the breadcrumb with props 1`] = `
<div
  className="partial-settled-payment-options"
>
  <h5
    className="card-title"
  >
    Partial settle payment
  </h5>
  Render links for partial refund
</div>
`;
